mypy/fscache.py

```diff
@@ -146,7 +146,7 @@ def _fake_init(self, path: str) -> os.stat_result:
         tpl = tuple(seq)
         # FIXME: this works around typeshed claiming stat_result is from posix
         # (typeshed #2683)
-        st = getattr(os, 'stat_result')(tpl)
+        st = getattr(os, 'stat_result')(tpl)  # noqa
```

Comment:
https://github.com/python/typeshed/pull/2683 was merged, so seems like this can just use `os.stat_result`?

In general I would have preferred for this to use `# type: ignore`, which is more greppable, and that way a run with `--warn-unused-ignores` could have found that this hack is no longer needed.

Reply:
Fixed. It was working around an issue that showed up in mypyc, not in mypy, so the getattr was needed for annoying reasons.